scripts/db_migrations/000002_create_tokens_table.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE projects ADD COLUMN IF NOT EXISTS token_ids text[];
```

Comment:
Are we just going with text instead of an explicit db link between the two tables for the sake of simplicity?

Reply:
Yes, more for simplicity, but I think linking (and another table) is the right thing to do.